rdr_service/dao/participant_dao.py

```diff
@@ -186,11 +186,12 @@ def _validate_update(self, session, obj, existing_obj):
                     base_name = "example"  # TODO: This is a hack because something sets up configs different
                     # when running all tests and it doesnt have the clientId key.
             base_name = base_name.lower()
```

Comment:
If not `base_name`, and `email` doesn't equal `DEV_MAIL`, this line is going to throw an exception.

Reply:
@j-kanuch cebyok does have a clientId set in the config, so there should be a base_name. I am not sure why this is going to throw an exception.